### ISSUE TITLE
Can't login via Scanning QR Code

### ISSUE THREAD

**Issue by anonymous0374** [nashaofu/dingtalk#271]
I can't login via scanning QR code.

Says: "Because of the update of terminal security protection of Alibaba Group and Ant Financial Group, the internal use of DingTalk’s web terminal had been taken offline. For any special needs, please contact Alime or IT service hotline for help."

So can't we login via Web / Electron version anymore? I'm a Ubuntu user.

Thanks.

**Comment by anonymous0374** [comment]
Release version: release-2.1.3
Desktop environment: Ubuntu 19.10

While I try login via webpage, the same error message pops up.
So I guess Alibaba has shut the API down?

**Comment by nashaofu** [comment]
it's work for me!You can try again at https://im.dingtalk.com/.

**Comment by anonymous0374** [comment]
that's strange, this is what I got
![error-login](https://user-images.githubusercontent.com/13930976/74585303-2c3e3b80-5016-11ea-8ead-dc865c248f41.png)

**Comment by anonymous0374** [comment]
Sorry the screenshot doesn't show properly.

I happened to freshly installed another laptop this morning(Ubuntu 18.04.4, Goolge Chrome latest, turned off ssr on both iPhone and laptop), tried to login via QR code.

The same error message(as above) pops up.

That sounds really wired.
Did you or anybody login via QR code on a Ubuntu machine(though I don't think it has anything to do with the OS, just curious)successfully?

Or maybe it has anything to do with my DingTalk account(which I did real-name verification already).
